src/commands/sync.js

```diff
@@ -21,6 +21,19 @@ module.exports = {
       abbr: 'ignore-hidden'
     },
     {
+      name: 'selectFromFile',
+      boolean: false,
+      default: false,
+      help: 'Sync only the list of selected files or directories in the given file. (default: .datdownload)',
```

Comment:
How do I use the default? I thought `dat sync --selectFromFile` would use `.datdownload` by default.